OxfordBodleian/Juel-Jensen/BDLaethf28.xml

```diff
@@ -96,9 +101,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                             </item>
                                         </list>
                                 </collation>
-                                
+                                <condition key="good">
+                                 The bookblock has been damaged by humidity, especially in the outer margins.
+                                 Hairs are still attached to <locus target="#49"/>.
+                                 Insect droppings are present on <locus target="#31 #32"/>. The blunt edge used for ruling has perforated the folio <locus target="#22"/>.
```

Comment:
Perhaps "the folio" is redundant. It should be generated automatically in the PDF from `locus target`